### ISSUE TITLE
Export error with InfluxDB 0.9

### ISSUE THREAD

**Issue by nicolargo** [nicolargo/glances#624]
The InfluxDB API has change:

- "name" => "measurement"
- "columns"/"points" => "fields"
-  ...

Before (InfluxDB v0.8):

```
[
  {
    "name" : "hd_used",
    "columns" : ["value", "host", "mount"],
    "points" : [
      [23.2, "serverA", "/mnt"]
    ]
  }
]
```

After (InfluxDB v0.9):

```
[
        {
            "measurement": "cpu_load_short",
            "tags": {
                "host": "server01",
                "region": "us-west"
            },
            "time": "2009-11-10T23:00:00Z",
            "fields": {
                "value": 0.64
            }
        }
    ]
```

See: https://github.com/influxdb/influxdb/issues/2564

**Comment by emidln** [comment]
I addressed this in: https://github.com/nicolargo/glances/pull/629

**Comment by emidln** [comment]
#629 was closed in favor of #630 which is properly rebased against develop.

**Comment by emidln** [comment]
This was merged into develop via #630 and can be closed.
